src/components/PermissionsDialog/PermissionsDialog.react.js

```diff
@@ -534,7 +534,7 @@ export default class PermissionsDialog extends React.Component {
       // hide suggestions to avoid ugly footer overlap
       this.refEntry.current.setHidden(hidden);
       // also show indicator when input is not visible
-      this.refScrollHint.current.toggleActive(hidden);
+      // this.refScrollHint.current.toggleActive(hidden);
```

Comment:
This line was throwing an error... the variable does not even exists... so I commented it out since I don't have enough context to fix it.